### PR TITLE
add notebook for Falcon

### PR DESCRIPTION
**Pull request by aniketmaurya**

### CODE REVIEW

**Review by carmocca — APPROVED** [review]
Looks good to me. It could have more hand-holding and text though. It's very barebones right now

File: notebooks/falcon-inference.ipynb [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "YJgQZwzi-UQ2"
+      },
+      "outputs": [],
+      "source": [
+        "# for CUDA\n",
+        "!pip install --index-url https://download.pytorch.org/whl/nightly/cu118 --pre 'torch>=2.1.0dev' -q"
```
[the remark this answers is not in the document]
Comment (aniketmaurya) [reply notebooks/falcon-inference.ipynb]: done

File: notebooks/falcon-inference.ipynb [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      "execution_count": null,
+      "metadata": {
+        "id": "11V8Aj7ZA6xb"
+      },
+      "outputs": [],
+      "source": [
+        "# install the dependencies\n",
+        "!pip install lightning\n",
+        "!pip install -r requirements.txt -q\n",
+        "!pip install huggingface_hub -q  # for downloading weights"
```
[the remark this answers is not in the document]
Comment (aniketmaurya) [reply notebooks/falcon-inference.ipynb]: done

File: notebooks/falcon-inference.ipynb
[the remark this answers is not in the document]
Comment (Andrei-Aksionov) [reply notebooks/falcon-inference.ipynb:1]: What about `Open in Colab` badge? Such a badge right at the top of the notebook is somewhat a standard nowadays.

-----
To test in PR:

[Open In Colab](https://colab.research.google.com/github/Lightning-AI/lit-parrot/blob/aniket/falcon-colab/notebooks/falcon-inference.ipynb)

```text
[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Lightning-AI/lit-parrot/blob/aniket/falcon-colab/notebooks/falcon-inference.ipynb)
```
----
This should be merged (link to main branch):

[Open In Colab](https://colab.research.google.com/github/Lightning-AI/lit-parrot/blob/main/notebooks/falcon-inference.ipynb)

```text
[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Lightning-AI/lit-parrot/blob/main/notebooks/falcon-inference.ipynb)
```

**Comment by lantiga** [comment]
@carmocca ok to merge?

**Pull request merged**